js/data/mysql_vms_single_az_template-1.4-100mb-dev.json

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "vm": "MySQL Server",
+        "instances": 3,
+        "vcpu": 2,
+        "ram": 8,
+        "ephemeral_disk": 30,
+        "persistent_disk": 1,
+        "dynamic_ips": 1,
+        "static_ips": 1,
+        "singleton": true
+    },
+    {
+        "vm": "Proxy",
+        "instances": 2,
+        "vcpu": 1,
+        "ram": 1,
+        "ephemeral_disk": 10,
+        "persistent_disk": 0,
+        "dynamic_ips": 1,
+        "static_ips": 1,
+        "singleton": true
+    },
+    {
+        "vm": "Broker",
+        "instances": 2,
+        "vcpu": 1,
+        "ram": 1,
+        "ephemeral_disk": 10,
+        "persistent_disk": 0,
+        "dynamic_ips": 1,
+        "static_ips": 1,
+        "singleton": true
+    },
+    {
+        "vm": "Broker Registrar",
+        "instances": 1,
+        "vcpu": 1,
+        "ram": 2,
+        "ephemeral_disk": 2,
+        "persistent_disk": 0,
+        "dynamic_ips": 0,
+        "static_ips": 2,
+        "singleton": true
+    },
+    {
+        "vm": "Broker De-Registrar",
+        "instances": 1,
+        "vcpu": 1,
+        "ram": 2,
+        "ephemeral_disk": 2,
+        "persistent_disk": 0,
+        "dynamic_ips": 0,
+        "static_ips": 2,
+        "singleton": true
+    },
+    {
+        "vm": "Compilation",
+        "instances": 1,
+        "vcpu": 4,
+        "ram": 4,
+        "ephemeral_disk": 20,
+        "persistent_disk": 0,
+        "dynamic_ips": 0,
+        "static_ips": 2,
+        "singleton": true
+    },
+    {
+        "vm": "Acceptance Test",
```

Comment:
@virtmerlin It seems logical to me to just reuse the VM's we size in the ERS for tile complication and errand VM's. Do you see any shortcomings with that approach? It makes life easier when integrating new services if we only have to worry about the service VM's.